sunpy/net/dataretriever/tests/test_fermi_gbm.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+
+from sunpy.time import parse_time
+from sunpy.time.timerange import TimeRange
+from sunpy.net.dataretriever.client import QueryResponse
+import sunpy.net.dataretriever.sources.fermi_gbm as fermi_gbm
+from sunpy.net.fido_factory import UnifiedResponse
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+from sunpy.net.tests.strategies import time_attr
+from hypothesis import given
+
+LCClient = fermi_gbm.GBMClient()
+
+
+@pytest.mark.remote_data
+@pytest.mark.parametrize("timerange,url_start,url_end",
```

Comment:
The indentation needs tidying up here.

Reply:
I agree, but it was done to make pep8 happy - suggested fix?

Reply:
Just try and align the different sections with their counterparts, i.e. the two `(` should be in the same columns. Also all the `'` should be in the same column as well.

Reply:
cool, I have done this now, but it gives pep8 problems - is that ok?

Thanks for all your help with this!

Reply:
It looks good enough :grin: